electroncash_plugins/fusion/plugin.py

```diff
@@ -688,10 +707,18 @@ def check_is_fuz_tx():
                     # Nope, lokad prefix not found
                     return False
                 # Step 2 - are at least 1 of the inputs from me? (DoS prevention measure)
+                fuz_input_found = False
                 for inp in inputs:
                     inp_addr = inp.get('address', None)
                     if inp_addr is not None and wallet.is_mine(inp_addr):
-                        return True  # Success! This tx matches our heuristics
+                        fuz_input_found = True
+                        if fuz_parents <= 0:
+                            return True # This transaction is a CashFusion tx
```

Comment:
Nit (my linter complains about such things although admitely the entire codebase has linting issues)

```suggestion
                            return True  # This transaction is a CashFusion tx
```